tests/test_vcs.py

```diff
@@ -186,3 +186,17 @@ def test_prompt_should_not_ask_if_no_input_and_rm_repo_dir(mocker, tmpdir):
 
     assert not mock_read_user.called
     assert not repo_dir.exists()
+
+
+def test_clone_should_raise_if_vcs_not_installed(mocker, tmpdir):
+    mocker.patch(
+        'cookiecutter.vcs.which',
+        autospec=True,
+        return_value=''
+    )
```

Comment:
I think this should be mocking `vcs.is_vcs_installed` rather than `witchcraft.which`?
And then add a `test_is_vcs_installed` that mocks `cookiecutter.vcs.which`?
This only raised my interest because `witchcraft.which` returns `None` if it can't find the executable, so we should probably change `cookiecutter.vcs.which` to test for `None` also?